deployment/cdk/opensearch-service-migration/README.md

```diff
@@ -89,13 +83,21 @@ cdk ls --c contextId=demo-deploy
 
 
 Depending on your use-case, you may choose to provide options from both the `cdk.context.json` and the CDK CLI, in which case it is important to know the precedence level for context values. The below order shows these levels with values being passed by the CDK CLI having the most importance
-1. CDK CLI passed context values (highest precedence)
+1. CDK CLI passed context values, e.g. --c stage=dev2 (highest precedence)
 2. Created `cdk.context.json` in the same directory as this README
 3. Existing `default-values.json` in the same directory as this README
 
-## Executing Commands on a Deployed Service
+## Which CDK context options should I use?
+See the [wiki - Configuration Options](https://github.com/opensearch-project/opensearch-migrations/wiki/Configuration-Options) for guidance on which context options should be used for different migrations.
+
+## How to use the deployed Migration tools?
+See the [wiki](https://github.com/opensearch-project/opensearch-migrations/wiki) for steps on how to use this tooling to perform different migrations.
 
-Once a service has been deployed, a command shell can be opened for that service's container. If the SSM Session Manager plugin is not installed, it should be installed when prompted from the below exec command.
+## Accessing the Migration Console Hub
```

Comment:
```suggestion
## Accessing the Migration Console
```

I don't think we call this a `hub` in another documentation, just the Migration Console or if we are getting _really_ verbose the Migration Assistance Console.